MAUI/Kanban-Board/migration.md

```diff
@@ -111,39 +111,11 @@ The following table illustrates the API migration for the kanban.
 </tr>
 </table>
 
-### KanbanPlaceholderStyle
+## Upcoming features in .NET MAUI
 
-<table>
-<tr>
-<tr>
-<th>Xamarin</th>
-<th>.NET MAUI</th></tr>
-</tr>
-<tr>
-<td>BackgroundColor</td>
-<td>Fill</td>
-</tr>
-<tr>
-<td>BorderColor</td>
-<td>Stroke</td>
-</tr>
-<tr>
-<td>BorderThickness</td>
-<td>StrokeWidth</td>
-</tr>
-<tr>
-<td>SelectedBackgroundColor</td>
-<td>SelectedFill</td>
-</tr>
-<tr>
-<td>SelectedBorderColor</td>
-<td>SelectedStroke</td>
-</tr>
-<tr>
-<td>SelectedBorderThickness</td>
-<td>SelectedStrokeWidth</td>
-</tr>
-</table>
+* Support for keeping cards.
```

Comment:
@EzhilarasanElangovan31 modify as - Support for keeping cards in the source column location during dragging until dropped.

Reply:
Modified